pocs/observatory.py

```diff
@@ -734,8 +736,20 @@ def _create_scheduler(self):
                 module = load_module(
                     'pocs.scheduler.{}'.format(scheduler_type))
 
+                obstruction_list = self.config['location'].get('obstructions', list())
```

Comment:
Is there a description of the values in the list somewhere?

Reply:
Just it the `Horizon` class. I haven't taken the step to add these to the config yet so will do that soon.

Reply:
Maybe a comment here pointing to the Horizon class?